Tests/UnitTest.xctestplan

```diff
@@ -137,8 +137,8 @@
     {
       "target" : {
         "containerPath" : "container:BrowserKit",
-        "identifier" : "LoggerTests",
-        "name" : "LoggerTests"
```

Comment:
If I get this right, this would mean LoggerTests wouldn't be part of the unit tests plan anymore 🤔 ? I think we should keep this

Reply:
The LoggerTest target doesn't exist anymore since it was moved into Common. This was just a leftover artifact that was never removed but wasn't causing any issues.

Reply:
Ah oups! Ok good